iac-extensions/kubernetes/src/test/resources/checks/MemoryLimitCheck/memory_limit_pod_with_global_limit.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: example
-  namespace: ${namespace}
+  namespace: with-global-limit
```

Comment:
In the `MemoryLimitCheckTest.testPodKindWithNamespace` this was being replaced by different namespaces. What's the reason for this change?

Reply:
I believe this is a mistake, I will remove this.